dependabot updates Tue May 10 11:02:39 PDT 2022

codeboten opened the pull request:
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /extension/observer/ecsobserver
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /exporter/kafkaexporter
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /exporter/awskinesisexporter
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /exporter/awsemfexporter
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /receiver/awscontainerinsightreceiver
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /processor/resourcedetectionprocessor
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /exporter/awsprometheusremotewriteexporter
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /exporter/datadogexporter
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /exporter/awscloudwatchlogsexporter
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /internal/aws/proxy
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /internal/aws/k8s
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /receiver/awsxrayreceiver
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /internal/aws/cwlogs
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /internal/aws/xray/testdata/sampleapp
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /internal/aws/xray
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /internal/aws/awsutil
Bump go.mongodb.org/atlas from 0.15.0 to 0.16.0 in /receiver/mongodbatlasreceiver
Bump github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/common from 1.0.394 to 1.0.395 in /exporter/tencentcloudlogserviceexporter
Bump google.golang.org/api from 0.78.0 to 0.79.0 in /exporter/f5cloudexporter
Bump google.golang.org/api from 0.78.0 to 0.79.0 in /exporter/googlecloudexporter
Bump google.golang.org/api from 0.78.0 to 0.79.0 in /exporter/googlecloudpubsubexporter
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.10 in /exporter/awsxrayexporter
Bump google.golang.org/api from 0.78.0 to 0.79.0 in /receiver/googlecloudpubsubreceiver
Bump google.golang.org/api from 0.78.0 to 0.79.0 in /receiver/googlecloudspannerreceiver

dmitryax commented:
@codeboten some tests are failing

bogdandrutu commented:
@codeboten would suggest to drop from this PR the upgrade of the `GoogleCloudPlatform` module.

Commit:
dependabot updates Tue May 10 12:36:32 PDT 2022
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /extension/observer/ecsobserver
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /exporter/kafkaexporter
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /exporter/awskinesisexporter
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /exporter/awsemfexporter
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /receiver/awscontainerinsightreceiver
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /processor/resourcedetectionprocessor
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /exporter/awsprometheusremotewriteexporter
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /exporter/datadogexporter
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /exporter/awscloudwatchlogsexporter
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /internal/aws/proxy
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /internal/aws/k8s
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /receiver/awsxrayreceiver
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /internal/aws/cwlogs
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /internal/aws/xray/testdata/sampleapp
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /internal/aws/xray
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.11 in /internal/aws/awsutil
Bump go.mongodb.org/atlas from 0.15.0 to 0.16.0 in /receiver/mongodbatlasreceiver
Bump github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/common from 1.0.394 to 1.0.395 in /exporter/tencentcloudlogserviceexporter
Bump google.golang.org/api from 0.78.0 to 0.79.0 in /exporter/f5cloudexporter
Bump google.golang.org/api from 0.78.0 to 0.79.0 in /exporter/googlecloudexporter
Bump google.golang.org/api from 0.78.0 to 0.79.0 in /exporter/googlecloudpubsubexporter
Bump github.com/aws/aws-sdk-go from 1.44.9 to 1.44.10 in /exporter/awsxrayexporter
Bump google.golang.org/api from 0.78.0 to 0.79.0 in /receiver/googlecloudpubsubreceiver
Bump google.golang.org/api from 0.78.0 to 0.79.0 in /receiver/googlecloudspannerreceiver
